eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -101,11 +101,16 @@ function RetrieveReleaseTag($artifactLocation, $continueOnError = $true) {
   }
 }
 
-function RetrievePackages($artifactLocation) {
-  $pkgs = Get-ChildItem -Path $artifactLocation -Include $packagePattern -Recurse -File
+function RetrievePackages($artifactLocation, [switch]$NoRecurse) {
```

Comment:
Can you remind me which part of our release steps use this method? Vs the discovery in the language settings?

Reply:
The origin is from https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/create-tags-and-git-release.ps1#L27 which is the core script that go uses to release.